packages/falter-profiles/files/etc/config/profile_petershagen_eggersdorf

```diff
@@ -0,0 +1,42 @@
+config 'community' 'profile'
+	option 'name' 'Freifunk Petershagen-Eggersdorf'
+	option 'homepage' 'https://freifunk.hacrafu.de'
+	option 'ssid' 'freifunk.hacrafu.de'
+	option 'ssid_scheme' 'addchannelbefore'
+	option 'mesh_network' '10.0.0.0/8'
+	option 'splash_network' '10.104.0.0/16'
+	option 'splash_prefix' '27'
+	option 'latitude' '52.536111'
+	option 'longitude' '13.805556'
+	option 'owm_api' 'http://util.berlin.freifunk.net'
+
+config 'defaults' 'wifi_device'
+	option 'channel' '13'
+
+config 'defaults' 'wifi_device_5'
+	option 'channel' '36'
+
+config 'defaults' 'wifi_iface'
+	option 'mcast_rate' '6000'
+
+config 'defaults' 'wifi_iface_5'
+	option 'mcast_rate' '12000'
+
+config 'defaults' 'bssidscheme'
+	option '10'	'02:CA:FF:EE:BA:BE'
+	option '13'	'D2:CA:FF:EE:BA:BE'
+	option '36'	'02:36:CA:FF:EE:EE'
+
+config 'defaults' 'ssidscheme'
+	option '13'	'intern-ch13.freifunk.net'
+	option '36'	'intern-ch36.freifunk.net'
+
+config 'defaults' 'wifi_iface_80211s'
+	option 'mesh_id' 'Mesh-Freifunk-Berlin'
```

Comment:
This one is only necessary, if you really want the mesh-Wifi to be called `Mesh-Freifunk-Berlin`. In other communities, it is plainly named `Mesh-Freifunk`, which will be also for you the case, if you remove this line.

If you want to, you could have a look at the profile of Fürstenwalde for reference:
https://github.com/freifunk-berlin/falter-packages/blob/master/packages/falter-profiles/files/etc/config/profile_fuerstenwalde